n26/core/templates/n26/trade_points.html

```
@@ -0,0 +1,161 @@
+                    </form>
+                </div>
+            </c-ui.card>
+        {% endif %}
+        <form method="post" action="{{ action }}">
+            {% csrf_token %}
+            <c-n26.form-section title="Start Visit Trading Post action"
+                                description="{% if receipt %}Starting another ends the one above, and its remaining Trade Points go with it.{% else %}Any fighter may go. A Leader brings 2 Trade Points and a Champion 1; the rest bring none, and one of them going is still what opens the post.{% endif %}">
+                {% if visitors %}
+                    <c-n26.tick-list :offer="offer" name="visiting" />
+                    <div>
+                        <c-ui.button type="submit" size="sm" variant="success">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)